CMakeLists.txt

```diff
@@ -235,7 +236,8 @@ ENDIF (NOT DEFINED LIB_DIR_SUFFIX)
 # set default include paths
 INCLUDE_DIRECTORIES(${PROJECT_SOURCE_DIR} ${Boost_INCLUDE_DIRS})
 
-# rpath handling
+# RPATH handling (see https://cmake.org/Wiki/CMake_RPATH_handling)
+# Note: RPATH only supported under Linux!
 SET(CMAKE_SKIP_BUILD_RPATH	FALSE)
```

Comment:
Hmm. I'm surprised. We shouldn't really be using RPATH, anyway.   RPATH is usually a hack intended for solving MacOS bugs.